docs/development/writingzeppelininterpreter.md

```diff
@@ -69,6 +69,9 @@ Here is an example of `interpreter-setting.json` on your own interpreter.
         "defaultValue": "property2DefaultValue",
         "description": "Property 2 description"
       }, ...
+    },
+    "editor": {
```

Comment:
Quick question - it's a bit not clear if this section is mandatory and if not, what is going to be the default one? May be it could be described in docs below. 
Asking as there are i.e PRs that implement a new interpreters, before this section was introduced.